### PR TITLE
feat: icq MaxKvQueryKeysCount and MaxTransactionsFilters gov gated #NTRN-364

### PR DESCRIPTION
**Pull request by swelf19**
The aim of the pr is to make ICQ module parameters governance gated, not hardcoded.

https://github.com/neutron-org/neutronjsplus/pull/50
https://github.com/neutron-org/neutronjs/pull/2
https://github.com/neutron-org/neutron-dev-contracts/pull/60
https://github.com/neutron-org/neutron-sdk/pull/158
https://github.com/neutron-org/neutron-integration-tests/pull/332
https://github.com/neutron-org/neutron-docs/pull/211

### CODE REVIEW

**Review by NeverHappened — APPROVED** [review]
please fix conflicts

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # cosmjs-types
```
Comment (sotnikov-s) [inline README.md:1]:
```suggestion
# neutronjs
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neutron-org/neutronjs",
-  "version": "0.1.0-rc4",
+  "version": "0.1.0-rc5",
   "description": "JS and TS types relating to Protocol Buffers used by Cosmos SDK and other related projects",
```
Comment (sotnikov-s) [inline package.json:4]:
```suggestion
  "description": "JS and TS types relating to Protocol Buffers used by Neutron",
```

File: .gitmodules [hunk 3]
```diff
@@ -16,6 +16,7 @@
 [submodule "neutron"]
 	path = protos/neutron-src
 	url = git@github.com:neutron-org/neutron.git
+  branch = feat/icq-params-gov-gated
```
Comment (sotnikov-s) [inline .gitmodules:19]:
```suggestion
[submodule "neutron"]
	path = protos/neutron-src
	url = git@github.com:neutron-org/neutron.git
        branch = feat/icq-params-gov-gated
```

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**